usaspending_api/api_contracts/contracts/v2/reporting/agencies/agency_code/differences.md

```diff
@@ -0,0 +1,89 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Agency Reporting Differences [/api/v2/reporting/agencies/{agency_code}/differences?{fiscal_year,fiscal_period,page,limit,order,sort}]
+
+This endpoint is used to power USAspending.gov's about the data obligation differences modal. This data can be used to better understand the way an agency submits data.
+
+## GET
+
+This endpoint returns an overview of government agency obligation differences data.
+
++ Parameters
+    + `agency_code`: `020` (required, string)
+        The specific agency code.
+    + `fiscal_year`: 2020 (required, number)
+        The fiscal year.
+    + `fiscal_period`: 10 (required, number)
+        The fiscal period.
+    + `page` (optional, number)
+        The page of results to return based on the limit.
+        + Default: 1
+    + `limit` (optional, number)
+        The number of results to include per page.
+        + Default: 10
+    + `order`: (optional, enum[string])
+        The direction (`asc` or `desc`) that the `sort` field will be sorted in.
+        + Default: `desc`
+        + Members
+            + `asc`
+            + `desc`
+    + `sort` (optional, enum[string])
+        A data field that will be used to sort the response array.
+        + Default: `difference`
+        + Members
+            + `difference`
+            + `file_a_obligations`
+            + `file_b_obligations`
+            + `tas`
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + `page_metadata` (required, PageMetaDataObject, fixed-type)
```

Comment:
Naming mismatch between `PageMetaDataObject` and `PageMetaData` defined below is throwing an error when attempting to run the mock API (similar in other files)

Reply:
resolved